src/core/control/Control.cpp

```diff
@@ -2130,59 +2130,82 @@ auto Control::openFile(fs::path filepath, int scrollToPage, bool forceOpen) -> b
     if ((loadedDocument != nullptr && loadHandler.isAttachedPdfMissing()) ||
         !loadHandler.getMissingPdfFilename().empty()) {
         // give the user a second chance to select a new PDF filepath, or to discard the PDF
-
         const fs::path missingFilePath = fs::path(loadHandler.getMissingPdfFilename());
 
-        std::string msg;
+        string parentFolderPath;
+        string filename;
+
+        // since POSIX systems detect the whole Windows path as a filename, this checks whether missingFilePath contains
+        // a Windows path
```

Comment:
I think you are right. It`s only necessary on POSIX systems, since windows can handle paths separated by '/' and '\'. Now there is only a small problem: on windows 'C:' gets prepended to any POSIX path, but this is only a visual nitpick, it does not harm the functionality.